05-mvc-with-rails/rails-sessions-and-auth-with-devise/readme.md

```diff
@@ -0,0 +1,319 @@
+# Devise! and sessions
+
+## Learning Objectives
+- explain how sessions give state to a web application
```

Comment:
Maybe "Explain what state is" too! It's a simple word that still trips up a lot of people.